recipes/wxpython/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wxPython" %}
+{% set version = "3.0.2.0" %}
+{% set sha256 = "d54129e5fbea4fb8091c87b2980760b72c22a386cb3b9dd2eebc928ef5e8df61" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   fn: wxPython-src-{{ version }}.tar.bz2  # [not win]
+   url: http://sourceforge.net/projects/wxpython/files/wxPython/{{ version }}/wxPython-src-{{ version }}.tar.bz2  # [not win]
+   sha256: {{ sha256 }}  # [not win]
+   fn: {{ version }}.zip  # [win]
+   url: https://github.com/NOAA-ORR-ERD/wxpython_wheels/archive/{{ version }}.zip  # [win]
```

Comment:
this is "common" and the amd54 one.

I suppose we could do the win32 one too, but I have no need for that.

This re-direction was a big 'ol pain -- but while lI found direct download urls from gohlke's site, for some reason conda build couldn't download them (my browser did fine).

and conda build also tried to unpack the wheels, rather than leaving them intact.

So putting them in a release on gitHub gave me a zip file with the wheels in it -- hacky, but it works.